Resources/doc/index.rst

```diff
@@ -12,8 +12,8 @@ a database, then you're in the right place.
 Prerequisites
 -------------
 
-This version of the bundle requires Symfony 2.1+. If you are using Symfony
-2.0.x, please use the 1.2.x releases of the bundle.
+This version of the bundle requires Symfony 3+. If you are using Symfony
```

Comment:
This is not true. The master branch is compatible with Symfony 2.3, 2.7, 2.8, 3.0 (and master I think).